typings/resin-sdk.d.ts

```diff
@@ -857,12 +880,81 @@ declare namespace ResinSdk {
 					): Promise<void>;
 					remove(uuidOrId: string | number, tagKey: string): Promise<void>;
 				};
+				configVar: {
+					getAllByDevice(
+						uuidOrId: string | number,
+						options?: PineOptionsFor<DeviceConfigVariable>,
+					): Promise<DeviceConfigVariable[]>;
+					set(
+						uuidOrId: string | number,
+						key: string,
+						value: string,
+					): Promise<void>;
+					get(
+						uuidOrId: string | number,
+						key: string,
+					): Promise<string | undefined>;
+					remove(
+						uuidOrId: string | number,
+						key: string,
+					): Promise<string | undefined>;
+				};
+				envVar: {
+					getAllByDevice(
+						uuidOrId: string | number,
+						options?: PineOptionsFor<DeviceEnvironmentVariable>,
+					): Promise<DeviceEnvironmentVariable[]>;
+					set(
+						uuidOrId: string | number,
+						key: string,
+						value: string,
+					): Promise<void>;
+					get(
+						uuidOrId: string | number,
+						key: string,
+					): Promise<string | undefined>;
+					remove(
+						uuidOrId: string | number,
+						key: string,
+					): Promise<string | undefined>;
+				};
+				serviceVar: {
+					getAllByDevice(
+						uuidOrId: string | number,
+						options?: PineOptionsFor<DeviceServiceEnvironmentVariable>,
+					): Promise<DeviceServiceEnvironmentVariable[]>;
+					set(
+						uuidOrId: string | number,
+						serviceId: number,
```

Comment:
For now, I don't want to. Service names can be a bit ambiguous (they're only unique by app, so really you need either a service id, or a app name/id + service name) and we don't actually need this for now. You're right, we can always add it later as a minor bump if we need it, and I'd rather ignore it for now.

Reply:
> so really you need either a service id, or a app name/id + service name

But we can extract the app from the device uuidOrId parameter, so that should be an issue, can't we?

> we can always add it later

:+1: